Src/DependencyCollector/Shared/Implementation/FrameworkHttpEventListener.cs

```diff
@@ -148,10 +147,10 @@ private void OnEndGetResponse(EventWrittenEventArgs eventData)
                         success = Convert.ToBoolean(eventData.Payload[1], CultureInfo.InvariantCulture);
                     }
 
-                    if (eventData.Payload[2] != null)
-                    {
-                        synchronous = Convert.ToBoolean(eventData.Payload[2], CultureInfo.InvariantCulture);
-                    }
+                    ////if (eventData.Payload[2] != null)
+                    ////{
+                    ////    synchronous = Convert.ToBoolean(eventData.Payload[2], CultureInfo.InvariantCulture);
+                    ////}
```

Comment:
Should just delete the commented out code since you can always get it back from history if need be.